src/adapters/scene.js

```diff
@@ -30,6 +30,7 @@ function Scene() {
 
 Scene.prototype.initMapBox = function () {
   this.mb = new Map({
+    attributionControl: false,
```

Comment:
I don't think this is used anywhere ?

Reply:
yep it is (if not there will be 2 attribute control, the native & the extended)